src/plugins/poster/poster.js

```diff
@@ -15,7 +15,8 @@ import $ from 'clappr-zepto'
 export default class PosterPlugin extends UIContainerPlugin {
   get name() { return 'poster' }
   get template() { return template(posterHTML) }
-  get shouldRender() { return this.container.playback.name !== 'html_img'}
+  get shouldRender() { return this.container.playback.name !== 'html_img' }
+  get shouldHideOnPlay() { return this.container.playback.name !== 'html_audio' }
```

Comment:
Ah yes I'd forgotten there's only the video hls playback.

Hls audio only playlists do exist though so maybe there should also be a HlsAudio playback? It gets complicated though because I think the extension would always be 'm3u8' so it would need more than an extension match :/

> On 20 Jan 2016, at 18:47, Mike Griffith <notifications@github.com> wrote:
> 
> In src/plugins/poster/poster.js:
> 
> > @@ -15,7 +15,8 @@ import $ from 'clappr-zepto'
> >  export default class PosterPlugin extends UIContainerPlugin {
> >    get name() { return 'poster' }
> >    get template() { return template(posterHTML) }
> > -  get shouldRender() { return this.container.playback.name !== 'html_img'}
> > +  get shouldRender() { return this.container.playback.name !== 'html_img' }
> > +  get shouldHideOnPlay() { return this.container.playback.name !== 'html_audio' }
> This isn't reliably working for me because I can't force HTML5Audio playback for a HLS audio-only stream. My last commit (manually specifying poster auto-hide behavior in options) worked fine for my use case, although this would be more elegant.
> 
> —
> Reply to this email directly or view it on GitHub.
>